src/battle_script_commands.c

```diff
@@ -7754,21 +7754,30 @@ static void Cmd_getmoneyreward(void)
 
     u32 money;
     u8 sPartyLevel = 1;
+    u8 b_buff_number = 5; 
 
     if (gBattleOutcome == B_OUTCOME_WON)
-    {
-        money = GetTrainerMoneyToGive(gTrainerBattleOpponent_A);
-        if (gBattleTypeFlags & BATTLE_TYPE_TWO_OPPONENTS)
-            money += GetTrainerMoneyToGive(gTrainerBattleOpponent_B);
-        AddMoney(&gSaveBlock1Ptr->money, money);
+    {        
+        if (VarGet(gSpecialVar_0x8003) == 1)
+        {
+            money = gSpecialVar_0x8004;
+            b_buff_number = 3; 
+            GiveFrontierBattlePoints();
+        }
+        else{
+            money = GetTrainerMoneyToGive(gTrainerBattleOpponent_A);
+            if (gBattleTypeFlags & BATTLE_TYPE_TWO_OPPONENTS)
+                money += GetTrainerMoneyToGive(gTrainerBattleOpponent_B);
+            AddMoney(&gSaveBlock1Ptr->money, money);
+        }
     }
     else
     {
         s32 i, count;
         for (i = 0; i < PARTY_SIZE; i++)
         {
             if (GetMonData(&gPlayerParty[i], MON_DATA_SPECIES_OR_EGG) != SPECIES_NONE
-             && GetMonData(&gPlayerParty[i], MON_DATA_SPECIES_OR_EGG) != SPECIES_EGG)
+            && GetMonData(&gPlayerParty[i], MON_DATA_SPECIES_OR_EGG) != SPECIES_EGG)
```

Comment:
the alignment here was correct. Please revert.

Reply:
I'd say the easiest way to review it, is to toggle the Frontier Pass in the Debug Menu under Flags&Vars.
![review](https://github.com/user-attachments/assets/1b0a4592-7111-4911-a94a-f29fa7bbdc12)